### PR TITLE
Generate Release Summary page

### PR DESCRIPTION
**Pull request by deepesh-verma**
- Read the output file in config
- Accept time range for releases
- Define custom structure for releases

Signed-off-by: deepesh-verma <deepeshvrm1@gmail.com>

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -6,3 +6,4 @@ organizations:
 days: 7
 filename: "pRs.json"
 prettyprintfile: "myfile.html"
+release-summary-filename: "release-summary.html"
```
Comment (arsulegai) [inline config.yaml:9]: [nitpick] Please also ignore this from getting copied during docker image creation and Git commit creation.

File: main.go [hunk 2]
```diff
@@ -102,3 +132,26 @@ func prettyPrint(expectedPrs []PullRequestDetails, fileName string) error {
 	err = t.Execute(f, expectedPrs)
 	return err
 }
+
+func saveIntoFileReleases(releaseList []ReleaseDetails, fileName string) error {
```
Comment (arsulegai) [inline main.go]: [nitpick] Could reuse the file save method, because it is doing json marshalling in both cases.

File: main.go [hunk 3]
```diff
@@ -102,3 +132,26 @@ func prettyPrint(expectedPrs []PullRequestDetails, fileName string) error {
...
+	fileContents, err := json.MarshalIndent(releaseList, "", "")
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(fileName, fileContents, 0644)
+}
+
+func prettyPrintReleases(releaseList []ReleaseDetails, fileName string) error {
```
Comment (arsulegai) [inline main.go]: [nitpick] There's scope of making this and PR print into one method with options. Not mandatory though.

**Review by arsulegai — COMMENTED** [review]
Please also raise it against the community resources repository

File: config.yaml (hunk 1 above)
Comment (deepesh-verma) [reply config.yaml:9]: Added the file to `.gitignore`.

File: main.go (hunk 2 above)
Comment (deepesh-verma) [reply main.go]: Added `report-utils.go` file, with the report generation related functions.

File: main.go (hunk 3 above)
Comment (deepesh-verma) [reply main.go]: Moved the print function to `report-utils.go`, and re-using it.

**Review by arsulegai — APPROVED** [review]

**Pull request merged**